src/api/interceptors/requestAuthCheck/requestAuthCheck.test.ts

```diff
@@ -0,0 +1,20 @@
+import { requestAuthCheck } from './requestAuthCheck';
+
+describe('requestAuthCheck function', () => {
+  it('adds Authorization prop to config headers', () => {
```

Comment:
prop jest w komponentach reactowych, a jak to się nazywa tutaj?

Reply:
header